fml/platform/win/file_win.cc

```diff
@@ -362,7 +362,10 @@ bool WriteAtomically(const fml::UniqueFD& base_directory,
       OpenFile(file_path.c_str(), true, FilePermission::kReadWrite);
 
   if (!temp_file.is_valid()) {
-    FML_DLOG(ERROR) << "Could not create temporary file.";
+    FML_DLOG(ERROR) << "Could not create temporary file: "
```

Comment:
A little confusing, but I think it's not the file that is temporary, but rather the file handle variable, so I think 'temporary' can be omitted from the error text.